src/Telemetry/Telemetry.cs

```diff
@@ -394,7 +394,6 @@ public enum TelemetryMeasureName
         GetColumnDefinitionsDurationMs,
         GetPrimaryKeysDurationMs,
         GetUnprocessedChangesDurationMs,
-        GetLockedRowCountDurationMs,
```

Comment:
The duration wasn't being calculated correctly so I just removed to keep things simpler. We haven't really used these durations for anything useful so far anyways - we can always add it back in if needed.